lib/nostrum/api.ex

```diff
@@ -734,15 +737,32 @@ defmodule Nostrum.Api do
   def bulk_delete_messages(channel_id, messages, filter \\ true)
 
   def bulk_delete_messages(channel_id, messages, false),
-    do: request(:post, Constants.channel_bulk_delete(channel_id), %{messages: messages})
+    do: send_chunked_delete(channel_id, messages)
 
   def bulk_delete_messages(channel_id, messages, true) do
     filtered_messages =
       Enum.filter(messages, fn message_id ->
```

Comment:
We could move this into the stream pipe in the `send_chunked_delete` function. This is something we should probably do, but will require you to rewrite these functions just a bit.

Reply:
Changed to take the messages as the first argument & pipe the messages in instead in f3d8061f4d75fa892046d273f19ed29624251c09.